+ JOB=8
+ NO_JOBS=16
+ [[ ! -n 8 ]]
+ [[ ! -n 16 ]]
+ [[ 8 == global ]]
+ set +x
+ pkgcheck -r gentoo --reporter FancyReporter 'net-misc/*' 'app-admin/*' 'app-emacs/*' 'x11-themes/*' 'net-dns/*' 'mail-client/*' 'dev-tcltk/*' 'app-benchmarks/*' 'sci-calculators/*' 'net-voip/*' -d imlate -d unstable_only -d cleanup -d stale_unstable -d UnusedGlobalFlags -d UnusedLicense --profile-disable-dev --profile-disable-exp
WARNING pkgcore: invalid glsa- glsa-201206-24.xml, package www-servers/tomcat:
               : error range rlt version 5.5.34 is a guaranteed empty set

net-misc/6tunnel
  DeprecatedEAPI: version 0.10: uses deprecated EAPI: 0
  DeprecatedEAPI: version 0.11_rc1: uses deprecated EAPI: 0
  MissingChksum: version 0.10: "6tunnel-0.10.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 0.11_rc1: "6tunnel-0.11rc1.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-misc/adjtimex
  DeprecatedEAPI: version 1.29-r1: uses deprecated EAPI: 2

net-misc/aget
  DeprecatedEAPI: version 0.4.1-r1: uses deprecated EAPI: 3
  MissingChksum: version 0.4.1-r1: "aget-0.4.1.tar.gz" missing required chksums: sha512, whirlpool; has chksums: rmd160, sha1, sha256, size

net-misc/aggregate
  DeprecatedEAPI: version 1.6: uses deprecated EAPI: 2
  MissingChksum: version 1.6: "aggregate-1.6.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-misc/aggregate-flim
  DeprecatedEAPI: version 1.0.2: uses deprecated EAPI: 0
  MissingChksum: version 1.0.2: "aggregate-1.0.2.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: [Errno 13] Permission denied
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 372, in _update_metadata
ERROR root:     with processor.reuse_or_request(ebp) as my_proc:
ERROR root:   File "/usr/lib64/python2.7/contextlib.py", line 17, in __enter__
ERROR root:     return self.gen.next()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 188, in reuse_or_request
ERROR root:     ebp = request_ebuild_processor(**request_kwds)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 71, in _inner
ERROR root:     return functor(*args, **kwds)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 141, in request_ebuild_processor
ERROR root:     e = EbuildProcessor(userpriv, sandbox, fakeroot, save_file)
ERROR root:   File "/usr/lib64/python2.7/site-packages/snakeoil/weakrefs.py",
          : line 155, in __call__
ERROR root:     instance = super(WeakRefFinalizer, cls).__call__(*a, **kw)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 362, in __init__
ERROR root:     os.setpgid(self.pid, 0)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: [Errno 13] Permission denied
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'

net-misc/arpd
  DeprecatedEAPI: version 0.2-r1: uses deprecated EAPI: 3
  MissingChksum: version 0.2-r1: "arpd-0.2.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-misc/arpsponge
  ExecutableFile: "files/arpsponge.initd" has unnecessary executable bit

net-misc/asterisk-core-sounds
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-en-alaw-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-en-g722-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-en-g729-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-en-gsm-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-en-siren7-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-en-siren14-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-en-sln16-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-en-ulaw-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-en-wav-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-fr-alaw-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-fr-g722-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-fr-g729-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-fr-gsm-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-fr-siren7-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-fr-siren14-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-fr-sln16-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-fr-ulaw-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-fr-wav-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-es-alaw-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-es-g722-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-es-g729-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-es-gsm-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-es-siren7-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-es-siren14-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-es-sln16-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-es-ulaw-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-es-wav-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-ru-alaw-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-ru-g722-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-ru-g729-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-ru-gsm-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-ru-siren7-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-ru-siren14-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-ru-sln16-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-ru-ulaw-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.21-r1: "asterisk-core-sounds-ru-wav-1.4.21.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-en-alaw-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-en-g722-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-en-g729-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-en-gsm-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-en-siren7-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-en-siren14-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-en-sln16-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-en-ulaw-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-en-wav-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-fr-alaw-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-fr-g722-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-fr-g729-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-fr-gsm-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-fr-siren7-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-fr-siren14-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-fr-sln16-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-fr-ulaw-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-fr-wav-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-es-alaw-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-es-g722-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-es-g729-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-es-gsm-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-es-siren7-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-es-siren14-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-es-sln16-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-es-ulaw-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-es-wav-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-ru-alaw-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-ru-g722-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-ru-g729-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-ru-gsm-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-ru-siren7-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-ru-siren14-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-ru-sln16-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-ru-ulaw-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.22: "asterisk-core-sounds-ru-wav-1.4.22.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-misc/asterisk-extra-sounds
  DeprecatedEAPI: version 1.4.11: uses deprecated EAPI: 2
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-en-alaw-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-en-g722-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-en-g729-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-en-gsm-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-en-siren7-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-en-siren14-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-en-sln16-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-en-ulaw-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-en-wav-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-fr-alaw-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-fr-g722-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-fr-g729-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-fr-gsm-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-fr-siren7-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-fr-siren14-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-fr-sln16-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-fr-ulaw-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 1.4.11: "asterisk-extra-sounds-fr-wav-1.4.11.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-misc/asterisk-moh-opsound
  DeprecatedEAPI: version 2.03: uses deprecated EAPI: 2
  MissingChksum: version 2.03: "asterisk-moh-opsound-alaw-2.03.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 2.03: "asterisk-moh-opsound-g722-2.03.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 2.03: "asterisk-moh-opsound-g729-2.03.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 2.03: "asterisk-moh-opsound-gsm-2.03.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 2.03: "asterisk-moh-opsound-siren7-2.03.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 2.03: "asterisk-moh-opsound-siren14-2.03.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 2.03: "asterisk-moh-opsound-sln16-2.03.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 2.03: "asterisk-moh-opsound-ulaw-2.03.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size
  MissingChksum: version 2.03: "asterisk-moh-opsound-wav-2.03.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-misc/asterisk-rate_engine
  DeprecatedEAPI: version 0.5.4: uses deprecated EAPI: 0
  MissingChksum: version 0.5.4: "rate-engine-0.5.4.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-misc/asterisk-spandsp_codec_g726
  DeprecatedEAPI: version 0.0.2_pre26: uses deprecated EAPI: 0
  MissingChksum: version 0.0.2_pre26: "spandsp-0.0.2pre26_codec_g726.c.bz2" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-misc/astmanproxy
  ExecutableFile: "files/astmanproxy.rc6" has unnecessary executable bit

net-misc/autossh
  MissingChksum: version 1.4c: "autossh-1.4c.tgz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-misc/axel
  DeprecatedEAPI: version 1.1: uses deprecated EAPI: 0
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
  DeprecatedEAPI: version 2.3-r1: uses deprecated EAPI: 0
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
  DeprecatedEAPI: version 2.4-r1: uses deprecated EAPI: 2
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 373, in _update_metadata
ERROR root:     mydata = my_proc.get_keys(pkg, self._ecache)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 835, in get_keys
ERROR root:     {"key": receive_key})
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py",
          : line 768, in _run_depend_like_phase
ERROR root:     self.write("%s %i\n%s" % (command, len(data), data),
          : append_newline=False)
ERROR root: MetadataException: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr keywords
ERROR root: error: Metadata Exception: pkg ebuild src:
          : sys-devel/gettext-0.19.4, attr data
ERROR root: error: bytes_ro_StringIO instance has no attribute '__len__'
  MissingChksum: version 2.3-r1: "axel-2.3.tar.bz2" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-misc/bfgminer
  NonExistentDeps: version 3.10.10: depset rdepends: nonexistent atoms [ dev-util/amd-app-sdk, dev-util/ati-stream-sdk, dev-util/ati-stream-sdk-bin, dev-util/amd-app-sdk-bin, dev-util/amdstream, dev-util/intel-opencl-sdk ]
  NonExistentDeps: version 4.10.3: depset rdepends: nonexistent atoms [ dev-util/amd-app-sdk, dev-util/ati-stream-sdk, dev-util/ati-stream-sdk-bin, dev-util/amd-app-sdk-bin, dev-util/amdstream, dev-util/intel-opencl-sdk ]
  NonExistentDeps: version 5.2.0: depset rdepends: nonexistent atoms [ dev-util/amd-app-sdk, dev-util/ati-stream-sdk, dev-util/ati-stream-sdk-bin, dev-util/amd-app-sdk-bin, dev-util/amdstream, dev-util/intel-opencl-sdk ]

net-misc/blinkperl
  DeprecatedEAPI: version 20030301-r1: uses deprecated EAPI: 0

net-misc/bopm
  DeprecatedEAPI: version 3.1.3: uses deprecated EAPI: 0
  MissingChksum: version 3.1.3: "bopm-3.1.3.tar.gz" missing required chksums: sha512, whirlpool; has chksums: sha256, size

net-misc/cadaver
  DeprecatedEAPI: version 0.23.3: uses deprecated EAPI: 3
ERROR pkgcore: caught unexpected exception accessing keywords from ebuild src:
             : sys-devel/gettext-0.19.4, exception Metadata Exception: pkg
             : ebuild src: sys-devel/gettext-0.19.4, attr keywords
ERROR pkgcore: error: Metadata Exception: pkg ebuild src:
             : sys-devel/gettext-0.19.4, attr data
ERROR pkgcore: error: bytes_ro_StringIO instance has no attribute '__len__'
ERROR pkgcore: None
ERROR root: check <pkgcheck.visibility.VisibilityReport object at
          : 0x7f9d716f10d0> raised
ERROR root: Traceback (most recent call last):
ERROR root:   File "/usr/lib64/python2.7/site-packages/pkgcheck/base.py", line
          : 337, in feed
ERROR root:     check.feed(item, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 264, in feed
ERROR root:     self.process_depset(pkg, attr, edepset, profiles, reporter)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in process_depset
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcheck/visibility.py", line
          : 315, in <genexpr>
ERROR root:     if any(True for pkg in src if visible(pkg)):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/restrictions/boolean.py
          : ", line 247, in match
ERROR root:     if not rest.match(vals):
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 101, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 207, in <lambda>
ERROR root:     s.data.pop("KEYWORDS", "").split()))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 103, in dynamic_getattr_dict
ERROR root:     raise_from(MetadataException(self, attr, str(e)))
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/base.py", line
          : 93, in dynamic_getattr_dict
ERROR root:     val = functor(self)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/package/metadata.py",
          : line 59, in _get_data
ERROR root:     return self._fetch_metadata()
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 274, in _fetch_metadata
ERROR root:     return self._parent._get_metadata(self, ebp=ebp,
          : force_regen=force_regen)
ERROR root:   File
          : "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py",
          : line 365, in _get_metadata
ERROR root:     return self._update_metadata(pkg, ebp=ebp)
ERROR root:   F